tools/python/boutiques/puller.py

```diff
@@ -11,19 +13,15 @@
     from urllib import urlretrieve
 
 
-class ZenodoError(Exception):
```

Comment:
what's the magic here, how can you raise a ZenodoError if the class was never defined?

Reply:
I noticed that `ZenodoError` was defined in 3 places so I removed it from `puller` and `searcher` and imported the one from `publisher`. So no magic there :)